### PR TITLE
Add uproot and it's dependencies

### PR DESCRIPTION
**Pull request by chrisburr**
Adds recipes for `uproot` and two of it's dependencies (`awkward` and `uproot-methods`).

### CODE REVIEW

File: recipes/awkward/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - numpy >=1.13.1
```
Comment (jakirkham) [inline recipes/awkward/meta.yaml]: Is this needed to build the recipe? If not, we should drop it from `build`.

File: recipes/uproot-methods/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy >=1.13.1
+    - awkward >=0.7.0
```
Comment (jakirkham) [inline recipes/uproot-methods/meta.yaml]: Are these needed to build the recipe? If not, we should drop them from `host`.

File: recipes/uproot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - python
+    - pip
+    - numpy >=1.13.1
+    - awkward >=0.7.0
+    - uproot-methods >=0.3.0
+    - cachetools
+    - backports.lzma  # [py27]
+    - lz4
+    # - xrootd
+    - pytest-runner
```
Comment (jakirkham) [inline recipes/uproot/meta.yaml]: Are these needed to build the recipe? If not, we should drop them from `host`.

**Comment by jakirkham** [comment]
Look pretty good. Some minor comments above. Should be straightforward to address. Please let me know if you have any questions. :)

**Comment by chrisburr** [comment]
@conda-forge/help-python Can someone take a look and merge this please?

**Comment by beckermr** [comment]
LGTM!

**Comment by chrisburr** [comment]
Thank you! 👍

**Pull request merged**